packages/core/database/package.json

```diff
@@ -39,6 +39,7 @@
     "lint": "run -T tsc --noEmit && run -T eslint ."
   },
   "dependencies": {
+    "@paralleldrive/cuid2": "2.2.2",
```

Comment:
nanoid library was only ESM compatible and I could not manage to import it in the database files

also, the main cuid library seems to be deprecated and its repo recommends this one as a replacement.

Reply:
I also took a look at ULID, which seems to be the most performant for inserts and sorts, but wanted to discuss with you first if you know alternatives or have any comment at all.